src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,227 @@
+
+_Components with common colors across both dark UI backgrounds_
+
+### High contrast
+
+In some cases, it is helpful to apply light components to dark backgrounds or light components to dark backgrounds. This technique is useful to focus attention or create visual tension when necessary.
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label=" ">
+        <img src="images/High_contrast_01.png" alt="High contrast example in a light theme.">    
+    </do-dont-example>
+    <do-dont-example correct=true label=" ">
+        <img src="images/High_contrast_02.png" alt="High contrast example in a dark theme.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+## Interaction states
+
+In addition to the enabled state tokens above, the five other primary interaction states have also been defined and tokenized for each theme. For values between Black and 70, interaction gets lighter and for values between 60 and White, interaction gets darker.
+
+#### Hover
+A unique set of hex values and color tokens exists for the hover states. These colors are outside of the core palette steps and can be considered as half steps in between two adjacent colors. In order to simplify our token structure, the White theme shares the same hover value as the Gray 10 theme; the two dark theme backgrounds share a hover value as well.
+
+#### Active
+The active state is two full steps lighter for values between 100 and 70 and two full steps darker for values between 60 and 10. Exception: White shares the same actived state as Gray 10 and Black shares the same active state as Gray 100. Example: the Blue 60 active state is Blue 80.
+
+#### Selected
+The selected state is one full step lighter for values between 100 and 70 and one full step darker for values between 60 and 10. Exception: White shares the same selected state as Gray 10 and Black shares the same selected state as Gray 100. Example: the Blue 60 selected state is Blue 70.
+
+#### Focus
+The focus state usually appears as a Blue 60 border in the light theme and as a White border in the dark theme.
+
+#### Disabled
+Disabled components are unavailable for interaction, so they don't receive hover or focus and are not subject to WC3 contrast compliance standards. For the dark themes, disabled values are two full steps lighter than their respective background. For light themes, two full steps darker. Exception: The White theme disabled value is Gray 30.     
+
+
+
+
+![Interactive states light theme](images/Interactive_states_light_new.png)
+
+_Light theme interactive tokens_
+
+
+![Interactive states dark theme](images/Interactive_states_dark_new.png)
+
+_Dark theme interactive tokens_
+
+
+## Digital Accessibility
+
+
+Using various forms of contrast is the most important consideration when making user-friendly color and interface choices. Awareness of standards and color blindness best practices is the key to accessible color selection.
+
+### WCAG 2.0 standards
```

Comment:
I think its WCAG AA not 2.0. There are 3 different level of accessibility, A, AA, AAA. We have to comply with the AA.

Reply:
We cover color a11y in detail in the a11y section. I would recommend removing this a11y content and linking to that page.

Reply:
Can we move this content to that page then? It doesn't have quite the same information and I think this explains it nicely and I like the visual aids.